### PR TITLE
feat: get users tasks & submitted transactions

### PR DESCRIPTION
**Pull request by cbrzn**

### CODE REVIEW

File: autotx/db.py [hunk 1]
```diff
@@ -288,7 +291,20 @@ def get_transactions(app_id: str, app_user_id: str, task_id: str, address: str,
         [TransactionBase(**tx) for tx in json.loads(result.data[0]["transactions"])], 
         result.data[0]["task_id"]    
     )
+
+def get_submitted_transactions_from_user(app_user_id: str) -> list[list[TransactionBase]]:
```
Comment (nerfZael) [inline autotx/db.py:295]: I'd add an app_id for more security

File: autotx/db.py [hunk 2]
```diff
@@ -154,6 +156,7 @@ def get_all(self) -> list[models.Task]:
             tasks.append(
                 models.Task(
                     id=task_data["id"],
+                    app_user_id=task_data["app_user_id"],
```
Comment (nerfZael) [inline autotx/db.py:159]: Why is this necessary? The apps know their user ids..

Comment (cbrzn) [reply autotx/db.py:159]: i needed the `app_user_id` to [filter the tasks of the user](https://github.com/agentcoinorg/AutoTx/blob/2641ad970ba2ac1169d41dcd797ab8d0e4eb5d05/autotx/server.py#L312) - is there other way to achieve that and maybe I am missing it?

Comment (nerfZael) [reply autotx/db.py:159]: You can add a new function to db to do that. It'll also be more performant instead of loading all of the tasks and filtering in memory. Also, it'll not expose the id to the app which I think this does.

**Review by nerfZael — APPROVED** [review]
